crates/sui-graphql-rpc/src/server/builder.rs

```diff
@@ -147,14 +155,17 @@ impl ServerBuilder {
                 .route("/graphql", post(graphql_handler))
                 .route("/health", axum::routing::get(health_checks))
                 .with_state(self.state.clone())
+                .route_layer(middleware::from_fn_with_state(
+                    self.state.clone(),
```

Comment:
It seems like you should be able to pass in just the `self.state.version` here, thereby avoiding cloning the entire app state and keeping it around just to get a reference to its version.

If you make the `Version` type `Copy` you can also avoid the explicit `clone()`.

Reply:
Thanks, I'll try that!